controls/1_3_master_node_controller_manager.rb

```diff
@@ -17,8 +17,10 @@
 
 title '1.3 Master Node: Controller Manager'
 
+controller_manager = attribute('controller_manager', default: kubernetes.hyperkube ? 'controller-manager' : 'kube-controller-manager', description: 'The name of the controller manager process')
```

Comment:
Wouldn't it be better to have a method called `kubernetes.controller_manager` where the method returns the right value?

Reply:
Good point! So I will then abstract the logic of the process name to return for the various components in the kubernetes class. The same applies to the next comment.